package.json

```diff
@@ -1,4 +1,5 @@
 {
+  "start": "webpack --config webpack.config.js --watch",
```

Comment:
- [ ] I can see that you added `"start": "webpack --config webpack.config.js --watch"` so that you can run `npm start` on your terminal as requested by the previous reviewer, but it should be added in the scripts. Kindly fix this. e.g
``` javascript
  "scripts": {
    "test": "echo \"Error: no test specified\" && exit 1",
   "start": "webpack --config webpack.config.js --watch"
    "watch": "webpack --watch",
    "build": "webpack"
  }
```